### PR TITLE
OKTA-819058 Acrolinx edits on Enterprise IdP page

### PR DESCRIPTION
**Pull request by thomascavanagh-okta**
## Description:
- **What's changed?** Improved the Acrolinx scores for the Enterprise IdP page
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-819058](https://oktainc.atlassian.net/browse/OKTA-819058)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appatidp.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fappatidp.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e323fa1b-edff-40fe-9e63-124c87137a11) | 38  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fappidpinokta.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8785259b-c740-4b65-bc54-48f8c7dc079b) | 21  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/createidpnote.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fcreateidpnote.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cbfad61a-dfa3-42e0-9f76-1ffcc1955865) | 25  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/659bf0b8-27dc-41d1-aaf4-1be6b8ab9dca) | 46  | 72  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appatidp.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappatidp.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/998cc2f6-b657-49a3-8c9d-c25f9e3299ea) | 36  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0cc46d96-c0b3-47b5-bb52-5e101780c5e2) | 63  | 78  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/afterappidpinokta.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fafterappidpinokta.md) | [97](https://okta.acrolinx.cloud/api/v1/checking/scorecards/90043736-8a92-44c3-a3d0-04e06556f6d6) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appatidp.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappatidp.md) | [92](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1c781ce6-2284-43ee-a0ec-f36e36164393) | 71  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5c87a5e2-431e-41f4-bd0e-d36cbd751839) | 58  | 83  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/accountlinking2.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Faccountlinking2.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0c5b830e-a419-4a43-94c2-d59c80def61e) | 100  | 50  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappatidp.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ce4d397e-7a78-41ce-8209-f0cfcb767210) | 17  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/73d5a609f43d570b95469c47605f6e690e0a955a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/42ae296e-fd72-40ca-b2d6-3ce54bca530c) | 44  | 68  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c1d5a1fb-b186-4d3e-b1dd-cc30993d22d1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appatidp.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 1. Create an Azure AD app using these [instructions](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appatidp.md:1]:
```suggestion
1. Register an [Azure app](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can try using this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
+> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can use this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
 
-Sometimes the issuer, single sign-on URL, and certificate aren't available from the external IdP until the metadata is uploaded to the external IdP. This metadata includes the Assertion Consumer Service URL (ACS URL) and the Audience URI. And, the ACS URL and Audience URI values aren't available until the IdP in Okta is configured. If that's the case with your external IdP, continue to the next step.
+When you set up an external IdP, sometimes the **Issuer**, **Single Sign-On URL**, and **Certificate** values aren't available. This information may not be available until the metadata is uploaded to the IdP. Also, the ACS URL and Audience URI values aren't available until you configure the IdP in Okta. If that's the case with your external IdP, continue to the [next step](#create-an-idp-in-okta).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:5]: I'm not sure if the user would "set up an external IdP". These instructions are for configuring an app (or registering an app) in the external IdP.

Can you move this information right after the "Use this information... " line?

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
 * **IdP Issuer URI**: The issuer URI of the IdP. This value is usually the SAML metadata `entityID` of the IdP `EntityDescriptor`.
 * **IdP Single Sign-On URL**: The binding specific IdP authentication request endpoint that receives SAML AuthN request messages from Okta.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:8]: On L9, change "Identity Provider" to "IdP".

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
-Access the external SAML Identity Provider (IdP) you want to integrate with and gather the following information. You use this information to configure the SAML Identity Provider in Okta in the next step. Make sure the SAML IdP supports Service Provider-initiated SAML.
+Access the external SAML IdP you want to integrate with and gather the following information. Use this information to configure the SAML IdP in Okta in the next step. Ensure the SAML IdP supports Service Provider-initiated SAML.
```
[the remark this answers is not in the document]
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:1]: Add this right after the bullet list:
```
Use this information to configure the SAML IdP in Okta in the next step. Ensure the SAML IdP supports SP-initiated SAML.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
 
-> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can try using this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
+> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can use this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:3]: Let's move this to the bottom of the snippet (L11). This is optional information that can be presented at the end.

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md (hunk 2 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:5]: Rewording suggestion:
```suggestion
If you don't have the issuer URI, SSO URL, and certificate values, continue to the [next step](#create-an-idp-in-okta) to generate the SAML metadata. The external IdP requires the ACS URI and audience URI, which are included in the generated metadata. Upload the metadata to your external IdP to obtain the issuer, single sign-on URL, and certificate values.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md [hunk 6]
```diff
@@ -28,8 +28,8 @@ Okta recommends that if the external IdP requires information from Okta for setu
 
 After you upload the metadata to the external IdP in the next step, you can edit the IdP in Okta. Then, you can enter the appropriate **IdP Issuer URI**, **IdP Single Sign-On URL**, and **Certificate** information:
 
-* **IdP Issuer URI**: The issuer. The Identity Provider provides this value.
+* **IdP Issuer URI**: The issuer. The IdP provides this value.
 
-* **IdP Single Sign-On URL**: The sign-on URL from the Identity Provider. If you sign the authN request by selecting **Request Signature**, but don't specify a **Destination**, Okta automatically sends the request to the IdP SSO URL.
+* **IdP Single Sign-On URL**: The sign-on URL from the IdP. You can sign the AuthN request by selecting **Request Signature**. If you don't specify a **Destination**, Okta automatically sends the request to the IdP SSO URL.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md:33]: You can use "authentication" for authN.
```suggestion
* **IdP Single Sign-On URL**: The sign-on URL from the IdP. You can sign the authentication request by selecting **Request Signature**. If you don't specify a **Destination**, Okta automatically sends the request to the IdP SSO URL.
```

**Review by vanngo-okta — COMMENTED** [review]
Thanks a lot for these improvements @thomascavanagh-okta! I made some structural/wording suggestions for saml2/appatidp, so hopefully that'll help you pass the 80 score threshold.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appatidp.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fappatidp.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f98c40ad-a654-41aa-96ea-119ca3fb48e5) | 41  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fappidpinokta.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1e72ba85-0a93-4d94-8a7f-ca1a9babf706) | 21  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/createidpnote.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fcreateidpnote.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0ab692cf-d9ed-48e4-a314-0ca2e753c867) | 25  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d07632d1-41e1-4402-8692-d3df6c25b4c9) | 46  | 72  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appatidp.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappatidp.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/149a17aa-2937-455b-8a5c-a8eade250033) | 36  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4c84b85b-e632-4820-b0fd-72c49f6326e3) | 63  | 78  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/afterappidpinokta.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fafterappidpinokta.md) | [97](https://okta.acrolinx.cloud/api/v1/checking/scorecards/65452d4a-d228-4428-b8d1-7d72b417ca38) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appatidp.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappatidp.md) | [92](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a13cf89e-8899-4dc0-899f-994ce3bf34f0) | 71  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8f74b3e9-2d45-4d7e-9fbc-b61476553896) | 58  | 83  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/accountlinking2.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Faccountlinking2.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e2c129f3-9007-4aaa-b7d8-43c41ad4af38) | 100  | 50  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappatidp.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/efbdea4e-bf88-4a5e-bedf-ceece424c05a) | 17  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/973b3d87e107143928078c4045a2cc5ea0da2857/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/89d9d96e-3288-45ce-b0ed-ba96695fe689) | 44  | 68  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.d44724a5-434a-4321-ab86-fb617fe717a8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appatidp.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fappatidp.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c05709bc-33bd-460e-a2a6-7ff6f855415e) | 38  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fappidpinokta.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dcd1faca-495a-4380-9139-924bd07c6983) | 21  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/createidpnote.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fcreateidpnote.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2f585ad5-15cb-4f31-932f-c8715336c303) | 25  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/13e57261-7534-4498-81f9-e81fe2385828) | 46  | 72  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appatidp.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappatidp.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ac84b9b4-7b8f-43aa-b4ee-491a1144b5f2) | 36  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7597cdba-964c-4c18-8280-e2f4531d6ec8) | 63  | 78  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/afterappidpinokta.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fafterappidpinokta.md) | [97](https://okta.acrolinx.cloud/api/v1/checking/scorecards/58c4d7d3-be3a-4d2d-bb1c-96a7563f1bf1) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appatidp.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappatidp.md) | [92](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3ac2b2d0-ce4a-4549-a05d-e3ef0ef2e927) | 71  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ca33fc1c-a028-4573-8d27-4de3ce0589cd) | 58  | 83  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/accountlinking2.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Faccountlinking2.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/98598927-21f2-46cd-bef9-5ec5dc5c9392) | 100  | 50  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappatidp.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0d07cca8-3b90-4026-bf3d-6a1a1cbeaa8e) | 16  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/8c7e8d8bc975b09b1662c985d47942f2d7e65be5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/52f39124-9e89-461a-9be0-6bfd8362cecb) | 44  | 68  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f73bea40-95e3-4000-bfa2-e630d45db3f6?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by brentschaus-okta — APPROVED** [review]
LGTM

**Pull request merged**